index.js

```diff
@@ -26,6 +30,32 @@ var new_client = function(api_key, config) {
     throw new Error("You must configure the client with an API key");
   }
 
+  if (client.stream) {
+    es = new EventSource(config.stream_uri + '/features', {headers: {'Authorization': 'api_key ' + client.api_key}});
+    client.features = {};
+
+    es.addEventListener('put', function(e) {
+      if (e.data) {
+        client.features = JSON.parse(e.data);
```

Comment:
Another thing we need to consider is how to block `toggle` calls until the `features` object has been populated. Otherwise, on initialization, end-users might receive the default values for a time.